cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In `TicketSerializer`, you are validating the ticket in the `validate` method. This is a good practice, but it would be better to move the validation logic to the `validate` method of the model. This way, the validation would be done at the model level and not at the serializer level. This separation of concerns makes your code more maintainable and testable.

Reply:
I use validate method in model and in serializer simultaneously, it's good practice